src/backend/base/langflow/components/data/api_request.py

```diff
@@ -25,59 +32,84 @@ class APIRequestComponent(Component):
         "This component allows you to make HTTP requests to one or more URLs. "
         "You can provide headers and body as either dictionaries or Data objects. "
         "Additionally, you can append query parameters to the URLs.\n\n"
-        "**Note:** Check advanced options for more settings."
+        "Note: Check advanced options for more settings."
     )
     icon = "Globe"
     name = "APIRequest"
 
+    default_keys = ["urls", "method", "query_params"]
+
     inputs = [
         MessageTextInput(
             name="urls",
             display_name="URLs",
             list=True,
             info="Enter one or more URLs, separated by commas.",
         ),
-        MessageTextInput(
-            name="curl",
-            display_name="cURL",
-            info="Paste a curl command to populate the fields. "
-            "This will fill in the dictionary fields for headers and body.",
-            advanced=False,
-            refresh_button=True,
-            real_time_refresh=True,
-            tool_mode=True,
-        ),
         DropdownInput(
             name="method",
             display_name="Method",
             options=["GET", "POST", "PATCH", "PUT"],
-            value="GET",
+            value="",
```

Comment:
What is the logic for not having the default as GET? Do we want that for sure?

Reply:
I faced some UI/UX issues with real_time_refresh = True, so I had to set the value property to 'GET' to ensure the component performs its initial refresh without breakning anything when loaded on the canvas